### PR TITLE
Minor edits

### PR DESCRIPTION
**Pull request by kessl**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -112,15 +112,13 @@ RQ: Do the certainty scores in CLIP3 predictions correspond to the actual catego
 ## Methods, results
 To answer the above, I examined the 'first-choice category' values in the primary data, focussing mainly on the highest and the lowest confidence values to scan the data for possible patterns. To interpret any tendencies found, I used preliminary evaluation data produced during the Hackathon ('Secondary data' above), and close-reading of the images themselves.
```
Comment (kessl) [inline README.md]: I just learned that *focussing* is a correct form and I hate it with a passion 🔥  I could get over the antiquated feel, but not over the hint of *cussing*

*From now on I'm only ever using ʙɪᴀꜱꜱɪɴɢ and ʙɪᴀꜱꜱᴇᴅ*

File: README.md [hunk 2]
```diff
@@ -64,27 +64,27 @@ Additionally, while CLIP1 and 2 both gave the top one classification prediction
...
 
 ### Secondary data
-(used for refining the RQ and hypotheses; would recommend for exploration of H2, H3)
 A spreadsheet containing (per every image from CLIP3 first-choice):
 - its URL (see Primary data)
 - CLIP3 first-choice category (see Primary data)
-- evaluation score of that classification (i.e., a percentage score expressing the probability of a human annotator classifying an image as belonging in the same first-choice category as CLIP3 did; based on human-annotated control batch produced by the group)<br>
+- evaluation score of that classification (i.e., a percentage score expressing the probability of a human annotator classifying an image as belonging in the same first-choice category as CLIP3 did; based on human-annotated control batch produced by the group)
+
 Various visualisations based on the above spreadsheet (created during the hackathon)
```
Comment (kessl) [inline README.md:83]: Hackathon is capitalized everywhere else, although personally I like it more lowercase

File: README.md [hunk 3]
```diff
@@ -64,27 +64,27 @@ Additionally, while CLIP1 and 2 both gave the top one classification prediction
...
 ### Secondary data
-(used for refining the RQ and hypotheses; would recommend for exploration of H2, H3)
 A spreadsheet containing (per every image from CLIP3 first-choice):
 - its URL (see Primary data)
 - CLIP3 first-choice category (see Primary data)
-- evaluation score of that classification (i.e., a percentage score expressing the probability of a human annotator classifying an image as belonging in the same first-choice category as CLIP3 did; based on human-annotated control batch produced by the group)<br>
-Various visualisations based on the above spreadsheet (created during the hackathon)
+- evaluation score of that classification (i.e., a percentage score expressing the probability of a human annotator classifying an image as belonging in the same first-choice category as CLIP3 did; based on human-annotated control batch produced by the group)
+
+Various visualisations based on the above spreadsheet (created during the Hackathon)
```
Comment (kessl) [inline README.md]: Hackathon is capitalized everywhere else, although personally I like it more lowercase

File: README.md [hunk 4]
```diff
@@ -149,18 +147,18 @@ Besides being a valuable item of information in and of itself, it allowed me to
 
 *Table 5 Detailed distribution of entries per certainty score range in the 'High-confidence' subset*
 
-Next (this further narrowing down of the subsets was mainly due to the high number of entires especially in the second subset), I took a random sample of approximately 270 entries (guided by: 10 % of 'Low-confidence' subset). The distribution across the 'Score ranges' (See Table 4 above) was weighted by the respective number of entries in these; i.e., for the 'Low-confidence' subset:
+Next (this further narrowing down of the subsets was mainly due to the high number of entries especially in the second subset), I took a random sample of approximately 270 entries (guided by: 10% of 'Low-confidence' subset). The distribution across the 'Score ranges' (See Table 4 above) was weighted by the respective number of entries in these; i.e., for the 'Low-confidence' subset:
```
Comment (kessl) [inline README.md]: I think it would be easier to read if the parenthetical clause was its own sentence

File: README.md [hunk 5]
```diff
@@ -64,27 +64,27 @@ Additionally, while CLIP1 and 2 both gave the top one classification prediction
...
+
+Various visualisations based on the above spreadsheet (created during the Hackathon)
 
 
 ## Research question(s), hypotheses
-RQ: Do the certainty scores in CLIP3 predictions correspond to the actual categories and/or visual form of the images classified, and, if so, then what are the tendencies? Is there significant variation in performance between clategories? Are there CLIP3 predictions not corresponding to human classification in cases classified by CLIP with low/high confidence? What can we conclude about the performance of a model thus customised?
+RQ: Do the certainty scores in CLIP3 predictions correspond to the actual categories and/or visual form of the images classified, and, if so, then what are the tendencies? Is there significant variation in performance between categories? Are there CLIP3 predictions not corresponding to human classification in cases classified by CLIP with low/high confidence? What can we conclude about the performance of a model thus customised?
 
 **H0:** There are no significant patterns in the performance of CLIP3 across certainty levels and categories.
```
Comment (kessl) [inline README.md:89]: does *fals.* really need to be shortened? (two lines below)

File: README.md [hunk 6]
```diff
@@ -355,25 +350,25 @@ In the whole examined sample, it is 331:479 of incorrect:correct (sum 3*270=810)
 ## Conclusions
```
Comment (kessl) [inline README.md:351]: The proportions above could also be expressed as percentages (% correct of total entries), for me that would be quicker to understand

(Github won't let me comment on some unchanged lines 🙈)

File: README.md [hunk 7]
```diff
@@ -355,25 +350,25 @@ In the whole examined sample, it is 331:479 of incorrect:correct (sum 3*270=810)
...
 
-**H2: CLIP3 performed significantly better in one or more categories (as compared to other categories).** and
+**H2: CLIP3 performed significantly better in one or more categories (as compared to other categories)**, and
 
-**H3: CLIP3 performed significantly worse in one or more categories (as compared to other categories).** had to be considered beyond the scope of this task. However, hopefully its results will provide useful background for anyone who might decide to examine them in the future (however, NB that the dataset itself is available upon request).
+**H3: CLIP3 performed significantly worse in one or more categories (as compared to other categories)** had to be considered beyond the scope of this task. However, hopefully its results will provide useful background for anyone who might decide to examine them in the future (however, NB that the dataset itself is available upon request).
```
Comment (kessl) [inline README.md]: what is NB?

File: README.md [hunk 8]
```diff
@@ -355,25 +350,25 @@ In the whole examined sample, it is 331:479 of incorrect:correct (sum 3*270=810)
...
 
-**H4a: A significant amount of first-choice classifications made by CLIP3 does not correspond to human classification in those cases where CLIP3 states low prediction confidence.** and
+**H4a: A significant amount of first-choice classifications made by CLIP3 does not correspond to human classification in those cases where CLIP3 states low prediction confidence**, and
 
-**H4b: A significant amount of second-choice classifications made by CLIP3 does not correspond to human classification in those cases where CLIP3 states low prediction confidence.** are both supported by this task's analysis, meaning that neither choice given was actually correct when its confidence score was low (Although NB the second-choice caveat as referred to also in HBb right below).
+**H4b: A significant amount of second-choice classifications made by CLIP3 does not correspond to human classification in those cases where CLIP3 states low prediction confidence** are both supported by this task's analysis, meaning that neither choice given was actually correct when its confidence score was low (Although NB the second-choice caveat as referred to also in HBb right below).
```
Comment (kessl) [inline README.md]: also NB + `HBb` may have been supposed to be `H5b`?

File: README.md [hunk 9]
```diff
@@ -382,7 +377,7 @@ Performing a similar mixed-methods analysis on the prediction results made by CL
 For the overview collected for H4a/b, it could be directly applicable to get more information on the patterns in the 0-1 category. Close-reading analysis could provide directions for specification alterations to get the result to 'flip' to 1-0 or 1-1. The correct second-choice classification might be a very useful lead for a relatively low-effort boost (cf. e.g., the 0-0 case).<br>
```
Comment (kessl) [inline README.md]: Even if I knew what `cf.` meant, two abbreviations in a row seem a bit much 😅

File: README.md [hunk 10]
```diff
@@ -382,7 +377,7 @@ Performing a similar mixed-methods analysis on the prediction results made by CL
...
+Based on observation from the close-reading analysis, it might be beneficial to redefine the 'Architecture' category. Images often fall between this and other categories (particularly 'Mathematics/Geometry/Astronomy' - building plan v 'generic' geometric drawing, as opposed to a hand-drawn column or building). One option might be to re-name the category and revise its keywords, so that it doesn't include any plans. Another one could be to split it between 'Mathematics/Geometry/Astronomy', 'Botany', 'Miscellaneous/Other', ...).
 
 As already mentioned in the analysis included in the H1 part, it might be useful to confirm or deny a (direct? exponential?) proportionality between a prediction's certainty score, and its actual correctness.
```
Comment (kessl) [inline README.md]: `direct? exponential?` 👀

File: README.md [hunk 11]
```diff
@@ -149,18 +147,18 @@ Besides being a valuable item of information in and of itself, it allowed me to
...
+| [20-25)     | 65                             |
+| [25-30)	  | 203                            |
 | **Σ**       | **270**                        | 
 
 *Table 6 Number of entries for analysis in the 'Low-confidence' subset*
 
-I close-read the randomly generated entries to verify whether the CLIP3 classification given for each of these corresponds to my own (i.e., human-produced) classification. I noted this down in a 0-1 system. I annotated both the first- and the second-choice classification in this manner, and the close-reading analysis results follow below. Table 7 below shows the four annotation possibilities and the actual distribution of these among the 270 entries annotated:<br>
+I close-read the randomly generated entries to verify whether the CLIP3 classification given for each of these corresponds to my own (i.e., human-produced) classification. I noted this down in a 0-1 system. I annotated both the first- and the second-choice classification in this manner, and the close-reading analysis results follow below. Table 7 shows the four annotation possibilities and the actual distribution of these among the 270 entries annotated:<br>
```
Comment (kessl) [inline README.md:162]: two *below*s

File: README.md [hunk 12]
```diff
@@ -112,15 +112,13 @@ RQ: Do the certainty scores in CLIP3 predictions correspond to the actual catego
 ## Methods, results
 To answer the above, I examined the 'first-choice category' values in the primary data, focussing mainly on the highest and the lowest confidence values to scan the data for possible patterns. To interpret any tendencies found, I used preliminary evaluation data produced during the Hackathon ('Secondary data' above), and close-reading of the images themselves. 
 
-To test H4a/b and H5a/b, I started by filtering out entries with the highest and the lowest first-choice confidence scores in the following ranges:<br>
+To test H4a/b and H5a/b, I started by filtering out entries with the highest and the lowest first-choice confidence scores in the following ranges:
+
 **'Low-confidence' subset**  (2734 entries)<br>
       x ∈ [0;30) <br>
-i.e.  0 ≤ x < 30.(0)
```
Comment (kessl) [inline README.md:118]: I think one of these is clear enough – I'd prefer `0.0 ≤ x < 30.0`, but I left the other one as that's the format you use in the tables

File: README.md (hunk 3 above)
Comment (lkalvoda) [reply README.md:83]:
```suggestion
Various visualisation on the above spreadsheet (created during the Hackathon)
```

File: README.md (hunk 5 above)
Comment (lkalvoda) [reply README.md:89]: I'd say _falsification_ -> keep

File: README.md (hunk 8 above)
Comment (lkalvoda) [reply README.md]: 🅱️

File: README.md [hunk 13]
```diff
@@ -63,28 +63,28 @@ These served as the training data for all three runs, with the second and third
...
 - the data from this run also provides the second most probable category the model assigned to each entry, which can shed light on the model's behaviour with this type of a dataset
 - the model used in the third run was already customised based on human-annotated feedback on the previous two
 
 ### Primary data
 A spreadsheet containing (per image):
 - its URL (from a database containing all images from the dataset; for qualitative close-reading analysis)
-- CLIP3 first-choice category and the certainty score of that classification (score is based on training data; for CLIP3, this is human-produced image classification annotations)
-- CLIP3 second-choice category and the certainty score of that classification (-,,-)
+- CLIP3 first-choice category and the certainty score of that classification (score is based on training data; for CLIP3, this is human-produced image classification annotations; sme for the second-choice)
```
Comment (lkalvoda) [inline README.md]:
```suggestion
- CLIP3 first-choice category and the certainty score of that classification (score is based on training data; for CLIP3, this is human-produced image classification annotations; same for the second-choice)
```

**Pull request merged**